lib/colorado_lottery.rb

```diff
@@ -0,0 +1,60 @@
+class ColoradoLottery
+
+  attr_reader :registered_contestants, :winners, :current_contestants
+  def initialize
+    @registered_contestants = Hash.new { |hash, key| hash[key] = [] }
```

Comment:
Great job with both of these hash initializations to give yourself the default value of a unique array for each key.